CHANGELOG.md

```diff
@@ -3,3 +3,18 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.1.0] - 2018-08-28
```

Comment:
Remove the version and the date until the moment you're actually going to tag and release it.